orchagent/portsorch.cpp

```diff
@@ -4236,6 +4217,36 @@ bool PortsOrch::addBridgePort(Port &port)
 
     if (port.m_bridge_port_id != SAI_NULL_OBJECT_ID)
     {
+        /* If the port is being added to the first VLAN,
+         * set bridge port admin status to UP.
```

Comment:
Won't the iterator in the remove_vlan member come back and delete this since it loops until last FDB is removed and then does remove_bridge_port?

Reply:
Agree with @dgsudharsan that there could be race condition since the `removeBridgePort `is not complete and the iterator is retried for removal after this addition. I think this logic is slightly complicated. Since the main motivation is to separate the vlan member map from the Port class, could you please make that change and get it merged for this release. We can have the bridge port handling as a different PR.